### PR TITLE
[Issue #3] Database deleting

### PR DESCRIPTION
**Pull request by piotrgos45**

### CODE REVIEW

File: .idea/deploymentTargetDropDown.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+          <Key>
+            <type value="VIRTUAL_DEVICE_PATH" />
+            <value value="C:\Users\pogo\.android\avd\Pixel_6_API_33.avd" />
+          </Key>
+        </deviceKey>
+      </Target>
+    </targetSelectedWithDropDown>
+    <timeTargetWasSelectedWithDropDown value="2022-12-08T11:05:47.402617300Z" />
+  </component>
+</project>
```
Comment (AlicjaKowalska) [inline .idea/deploymentTargetDropDown.xml:17]: This file is unnecessary. Maybe we should add it to .gitignore

**Review by AlicjaKowalska — APPROVED** [review]

File: .idea/deploymentTargetDropDown.xml (hunk 1 above)
Comment (piotrgos45) [reply .idea/deploymentTargetDropDown.xml:17]: done :D

File: .gitignore [hunk 2]
```diff
@@ -7,6 +7,7 @@
 /.idea/workspace.xml
 /.idea/navEditor.xml
 /.idea/assetWizardSettings.xml
+/.idea/deploymentTargetDropDown.xml
```
Comment (michalmobi) [inline .gitignore:10]: try this:
`.idea/*`
it should ignore all files in idea folder

Comment (piotrgos45) [reply .gitignore]: Good point!

**Review by kozych — COMMENTED** [review]
Please resolve conflicts (rebase databaseDeleting onto main)

**Review by kozych — APPROVED** [review]

**Review by kozych — CHANGES_REQUESTED** [review]
Please resolve conflicts (rebase databaseDeleting onto main)

**Review by kozych — APPROVED** [review]

File: app/src/main/java/com/example/patientassistant/Room/AppointmentDAO.kt [hunk 3]
```diff
@@ -16,4 +16,9 @@ interface AppointmentDAO {
     @Update
     suspend fun update(appointment: Appointment)
 
+    @Delete
+    suspend fun delete(appointment: Appointment)
+
+    @Query("DELETE FROM appointment_table")
```
Comment (michalmobi) [inline app/src/main/java/com/example/patientassistant/Room/AppointmentDAO.kt:22]: please create constants for table names

File: app/src/main/java/com/example/patientassistant/Room/DrugDAO.kt [hunk 4]
```diff
@@ -14,4 +16,10 @@ interface DrugDAO {
     @Update
     suspend fun update(drug : Drug)
 
+    @Delete
+    suspend fun delete(drug: Drug)
+
+    @Query ("DELETE FROM drug_table")
```
Comment (michalmobi) [inline app/src/main/java/com/example/patientassistant/Room/DrugDAO.kt:22]: please create constants for table names

**Review by michalmobi — CHANGES_REQUESTED** [review]

**Review by kozych — APPROVED** [review]

File: app/src/main/java/com/example/patientassistant/Objects/TableNames.kt [hunk 5]
```diff
@@ -0,0 +1,6 @@
+package com.example.patientassistant.Objects
+
+object TableNames {
```
Comment (michalmobi) [inline app/src/main/java/com/example/patientassistant/Objects/TableNames.kt:3]: this is fine for now, but in general I recommend creating companion objects, not general objects.

**Review by michalmobi — APPROVED** [review]

**Pull request merged**